### PR TITLE
Chef 14 compatibility fixes

### PR DESCRIPTION
**Pull request by khashf**
**Old ChefDK part 1**
- [x] Ensure all ChefSpec tests pass
- [x] Ensure all Inspec tests pass

**New ChefDK**
- [x] Ensure cookstyle is passing
- [x] Apply foodcritic fixes
- [x] Ensure running rake passes
- [x] Ensure all Inspec tests pass

**Old ChefDK part 2**
- [x] Repeat all tests above using old ChefDK

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@
 depends          'hostsfile'
 depends          'openssl'
 depends          'poise-python'
-depends          'runit'
+depends          'runit', '~> 4.3.1'
```
Comment (ramereth) [inline metadata.rb]: Why pin this version?

Comment (khashf) [reply metadata.rb]: `runit` from 5.0.0 and above requires at least chef 14. This will allow this cookbook to be backward-compatible with chef 13

Comment (ramereth) [reply metadata.rb]: Can you please add a comment to the right which states that?

File: recipes/default.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@
 end
 
 include_recipe 'git'
-include_recipe 'build-essential::default'
+build_essential 'ganeti_webmgr'
```
Comment (ramereth) [inline recipes/default.rb]: Can you please add a ChefSpec test for this?

Comment (khashf) [reply recipes/default.rb]: a49d3a6

File: metadata.rb (hunk 1 above)
Comment (khashf) [reply metadata.rb]: 7f1f06c

**Comment by ramereth** [comment]
@khashf I pushed some fixes to get it going. There were two problems:
1. You were missing additional cookbooks to resolve in Berksfile
2. The key problem with percona not getting setup was that the VM didn't have enough RAM. I bumped up the size to deal with that.

There's still an issue but I'll leave you to fix it :)

**Comment by ramereth** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v1.1.0.  Have a nice day!

**Pull request merged**